onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -97,6 +97,7 @@ namespace perftest {
       "\t    [QNN only] [device_id]: The ID of the device to use when setting 'htp_arch'. Defaults to '0' (for single device). \n"
       "\t    [QNN only] [enable_htp_fp16_precision]: Enable the HTP_FP16 precision so that the float32 model will be inferenced with fp16 precision. \n"
       "\t    Otherwise, it will be fp32 precision. Only works for float32 model. Defaults to '0' (with FP32 precision.). \n"
+      "\t    [QNN only] [context_node_name_prefix]: QNN EPContext node name prefix to make sure the generated node name unique for split models. e.g 'phi3_part1'.\n"
```

Comment:
Does this also need to be added to the onnx_test_runner tool? Probably not, but thought I would make sure.

Reply:
I realize that use perf_test tool is more convenient most of the time since no need to prepare the input/output.pb.